sonic_data_client/mixed_db_client.go

```diff
@@ -75,6 +75,58 @@ type MixedDbClient struct {
 
 var mixedDbClientMap = map[string]MixedDbClient{}
 
+func hget(table swsscommon.Table, key string, field string) (string, error) {
+	var fieldValuePairs = swsscommon.NewFieldValuePairs()
```

Comment:
Now we are using redis api to read from redis server, do you think we should use this new API?

Reply:
I think it's not necessary to change other Redis code, there are too much code, the code here using swsscommon because ZMQ related code already using swsscommon.